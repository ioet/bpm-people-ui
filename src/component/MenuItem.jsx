import React from 'react';
import PropTypes from 'prop-types';
import styles from './generalStyle.css';

const MenuItem = ({ text }) => (
  <div className={styles.menuItemStyle}>
    {text}
  </div>
);

MenuItem.propTypes = {
  text: PropTypes.string,
};

MenuItem.defaultProps = {
  text: 'item',
};

export default MenuItem;
