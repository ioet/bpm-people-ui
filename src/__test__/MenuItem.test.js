import React from 'react';
import MenuItem from '../component/MenuItem';

describe('<MenuItem/>', () => {
  test('Component Shallow Test', () => {
    global.shallow(<MenuItem />);
  });
});
