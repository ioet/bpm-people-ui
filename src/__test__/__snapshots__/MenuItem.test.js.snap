// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem/> Component renders default value for prop 1`] = `
<div
  className="menuItemStyle"
>
  item
</div>
`;
